tasks/main.yml

```diff
@@ -90,6 +65,8 @@
     priv: "{{ database_name }}.*:ALL,GRANT"
     login_user: "{{ database_root_user }}"
     login_password: "{{ database_root_password }}"
+    check_implicit_admin: true
+    login_unix_socket: /run/mysqld/mysqld.sock
```

Comment:
This needs a os-specific variable as well. E.g. on CentOS Stream 9 the defautl socket location is `/var/lib/mysql/mysql.sock`.